src/components/support-form/steps/GeneralInfo.tsx

```diff
@@ -89,6 +97,28 @@ export default function GeneralInfo({ formik }: GeneralInfoProps) {
           onChange={formik.handleChange}
         />
       </Grid>
+      <Grid container direction="column" alignItems="flex-end">
```

Comment:
Maybe we can make the checkbox with full width of the form, so it's not right aligned.

This will help us avoid breaking the UI if the translated text is longer